pkg/controllers/provisioning/v1alpha1/reallocation/utilization.go

```diff
@@ -125,20 +123,15 @@ func (u *Utilization) clearUnderutilized(ctx context.Context, provisioner *v1alp
 // markTerminable checks if a node is past its ttl and marks it
 func (u *Utilization) markTerminable(ctx context.Context, provisioner *v1alpha1.Provisioner) error {
```

Comment:
Should this function name be changed since it now deletes rather than just marking the node? Or is the action the same with the finalizer?

Reply:
Good call. Thanks!